pkg/dotc1z/sync_runs.go

```diff
@@ -303,32 +303,49 @@ func (c *C1File) StartSync(ctx context.Context) (string, bool, error) {
 	if sync != nil && sync.EndedAt == nil {
 		syncID = sync.ID
 	} else {
-		q := c.db.Insert(syncRuns.Name())
-		q = q.Rows(goqu.Record{
-			"sync_id":    syncID,
-			"started_at": time.Now().Format("2006-01-02 15:04:05.999999999"),
-			"sync_token": "",
-		})
-
-		query, args, err := q.ToSQL()
-		if err != nil {
-			return "", false, err
-		}
-
-		_, err = c.db.ExecContext(ctx, query, args...)
+		syncID, err = c.StartNewSync(ctx)
```

Comment:
I think syncID's declaration shouldn't initialize. It should just be `var syncID string`. We either set it to sync.ID or we set it with the return value from StartNewSync() here.